examples/CMakeLists.txt

```
@@ -63,3 +63,18 @@ if (BLACKMAGIC_FOUND)
 	# Link test executable to the new library
 	target_link_libraries(openshot-blackmagic openshot)
 endif()
+
+############### OPENCV EXAMPLE ################
+find_package( OpenCV 4 )
+if (OpenCV_FOUND)
+	# Create test executable
+	add_executable(openshot-example-opencv
+			Example_opencv.cpp)
+	
+	target_compile_definitions(openshot-example-opencv PRIVATE
+	-DTEST_MEDIA_PATH="${TEST_MEDIA_PATH}" )
+
+	# Link test executable to the new library
+	target_link_libraries(openshot-example-opencv openshot)
+
+endif()
```

```suggestion

############### OPENCV EXAMPLE ################
if (DEFINED CACHE{HAVE_OPENCV})
	# Create test executable
	add_executable(openshot-example-opencv
			Example_opencv.cpp)
	
	target_compile_definitions(openshot-example-opencv PRIVATE
	-DTEST_MEDIA_PATH="${TEST_MEDIA_PATH}" )

	# Link test executable to the new library
	target_link_libraries(openshot-example-opencv openshot)
endif()
```

`src/CMakeLists.txt` sets `HAVE_OPENCV` to `TRUE` as a `CACHE` variable, when it imports the OpenCV targets. So it's usable at every level of the build, and this way you're depending on whether libopenshot was actually configured to use OpenCV.